lib/howl/commands/edit_commands.moon

```diff
@@ -98,6 +98,25 @@ command.register
       app.editor.line_at_top = replacement.line_at_top
 
 command.register
+  name: 'buffer-sort-lines'
+  description: 'Sort the selected lines or the entire buffer'
+  handler: ->
+    editor = howl.app.editor
+    local lines
```

Comment:
I have strong Python habits so the if expression looks odd to me too, specially since it's on multiple lines.